### PR TITLE
Bug 1930106: bump ignition to v3_2

### PR DESCRIPTION
**Pull request by yuvalk**
With tests modified accordingly

this is a manual cherry-pick from 0920b5b904ac8b346fc117b4ef3b7e88ba2c2bb4

### CODE REVIEW

**Comment by stbenjam** [comment]
/retitle Bug 1930106: bump ignition to v3_2

File: go.mod [hunk 1]
```diff
@@ -20,8 +20,8 @@ require (
 	github.com/c4milo/gotoolkit v0.0.0-20190525173301-67483a18c17a // indirect
 	github.com/clarketm/json v1.14.1
 	github.com/containers/image v3.0.2+incompatible
-	github.com/coreos/ignition/v2 v2.3.0
-	github.com/dmacvicar/terraform-provider-libvirt v0.6.2
+	github.com/coreos/ignition/v2 v2.9.0
+	github.com/dmacvicar/terraform-provider-libvirt v0.6.4-0.20201216193629-2b60d7626ff8
```
Comment (staebler) [inline go.mod:24]: Why does the libvirt terraform provider need to be updated?

Comment (yuvalk) [reply go.mod]: thanks! that probably slipped through the backport

**Comment by staebler** [comment]
The following tests should not be required. They were made required incorrectly when copied from the master pre-submits.

/override ci/prow/e2e-kubevirt
/override ci/prow/e2e-metal-assisted
/override ci/prow/e2e-openstack-ipi

**Comment by staebler** [comment]
The following tests should not be required. They were made required incorrectly when copied from the master pre-submits.

/override ci/prow/e2e-kubevirt
/override ci/prow/e2e-metal-assisted
/override ci/prow/e2e-openstack-ipi
/override ci/prow/e2e-metal-ipi

**Pull request merged**